### ISSUE TITLE
Aria-Label missing for various buttons

### ISSUE THREAD

**Issue by Simon-Laux** [deltachat/deltachat-desktop#4611]
- [ ] jump down button in message list (both modes should have different strings (jump back, jump/scroll down completely), translation strings also need to be create)
- [ ] verified chat/contact icon has no aria label
- [ ] accounts in account list should have account name & label in aria label -> maybe also connection state and if account is muted

**Comment by WofWca** [comment]
To be precise, not all of this needs to be fixed by adding an `aria-label` attribute. For example, there is `aria-description`.

For account list, maybe `aria-describedby` is more suitable because we have a tooltip / popup there.

Also, while we're at it, some `aria-label`s are English-only currently. They can be found with `aria-label='` and `aria-label={'`.

About avatars, here are the places where it's missing:

https://github.com/deltachat/deltachat-desktop/blob/48e755d88bd2c3ba8f94fd9e5fe878fea84249e8/packages/frontend/src/components/Avatar/index.tsx#L111

https://github.com/deltachat/deltachat-desktop/blob/a5c02433976909fe01a64f3049c96d0d101ae9d8/packages/frontend/src/components/GroupImage.tsx#L49

Update:
- [x] "cancel reply" is just "close"
  https://github.com/deltachat/deltachat-desktop/blob/5329b5c68a20136705bf4c0bac2066c7c263f65e/packages/frontend/src/components/composer/Composer.tsx#L376
- [x] the "edit profile" button's label is not "edit profile": https://github.com/deltachat/deltachat-desktop/pull/5799#pullrequestreview-3531248129.

**Comment by nicodh** [comment]
> About avatars, here are the places where it's missing:

Wouldn't it make more sense to set them to aria-hidden="true" since there is no real information inside the avatar and the displayname is already presented as text

**Comment by WofWca** [comment]
Yes, that's what I did for some of the avatars. Although this doesn't apply to _all_ avatars, e.g. the ones in the "View Profile" dialog.

**Comment by nicodh** [comment]
Why - in the profile the displayname is shown below the avatar image. So what additional information is in the avatar image?

**Comment by WofWca** [comment]
Well at least because the avatar is clickable (and the profile name click doesn't perform the same action), where you can do stuff with the image: copy, save, screenshot, show to others. AFAIK it doesn't make sense for something to be both focusable and `aria-hidden` at the same time.

**Comment by r10s** [comment]
accessibility has hugely improved the last time, and we will continue to do things here, same as on android/iOS that are praised a lot for its accessibility.

however, we also need to take care to not spend too many of the limited (discussion) resources here and lose focus. at some point, we might do a boost, until then, these discussions should go to the support forum, and are in triage with lots of other things
